packages/marathon/build

```diff
@@ -19,6 +19,8 @@ StartLimitInterval=0
 RestartSec=15
 PermissionsStartOnly=True
 EnvironmentFile=/opt/mesosphere/environment
+EnvironmentFile=-/opt/mesosphere/etc/marathon
```

Comment:
https://mesosphere.github.io/marathon/docs/command-line-flags.html

Note - Command Line Flags May Be Specified Using Environment Variables

Seems to suggest otherwise. We do use cli flags currently, should probably move